hasher-matcher-actioner/hmalib/common/configs/tx_apis.py

```diff
@@ -68,14 +74,15 @@ def add_signal_exchange_api(
     return AddSignalExchangeAPIResult.ADDED
 
 
-class DisableSignalExchangeAPIResult(Enum):
+class UpdateSignalExchangeAPIStatusResult(Enum):
     DISABLED = 1
-    FAILED = 2
+    ENABLED = 2
+    FAILED = 3
 
 
-def disable_signal_exchange_api(
-    klass: str,
-) -> DisableSignalExchangeAPIResult:
+def set_status_signal_exchange_api(
+    klass: str, status: bool
```

Comment:
Should this be changed to take the enum instead of the status?

Reply:
Yeah. I guess that would be more fool-proof. Will fix.